src/dash-table/components/Table/props.ts

```diff
@@ -410,6 +411,7 @@ export type SanitizedProps = Omit<Omit<
     Merge<PropsWithDefaults, {
         fixed_columns: number;
         fixed_rows: number;
+        loading_state: boolean;
```

Comment:
Synthetize the `loading_state` object to a boolean representing whether `data` is being worked on. Prevents a bunch of useless re-renders.